### ISSUE TITLE
ProcessTimedOutException on long running PHPUnit tests

### ISSUE THREAD

**Issue by lasselehtinen** [phpro/grumphp#149]
Hi,

I have long running integration tests which occasionally take longer than one minute. Is there any way to configure the timeout to be longer than the default 60 seconds?

```
GrumPHP detected a pre-commit command.
GrumPHP is sniffing your code!
Running task 1/5: Composer
Running task 2/5: Blacklist
Running task 3/5: Phpcs
Running task 4/5: Phpunit

[Symfony\Component\Process\Exception\ProcessTimedOutException]
The process "'./vendor/bin/phpunit'" exceeded the timeout of 60 seconds.
```

**Comment by mikechernev** [comment]
@lasselehtinen Maybe it's better if you remove the limit in your PHP config for the CLI, rather then doing this in the code.
All you have to do is set the `max_execution_time` property to `-1` in your php.ini.

I hope this helps 😉

**Comment by veewee** [comment]
Thanks for reporting!
By default the Symfony process component uses a timeout of 60s.
It can be incremented or set to `null` to make sure the process isn't terminated.
Currently this is not configurable in grumphp. I'll make an additional parameter for this.

**Comment by veewee** [comment]
This one is fixed in #151.

**Comment by lasselehtinen** [comment]
Excellent, thanks.
